packages/clay-navigation-bar/src/ClayNavigationBar.js

```diff
@@ -26,6 +26,7 @@ class ClayNavigationBar extends ClayComponent {
 		if (this._isTransitioning && !this._visible) {
 			this._setCollapseHeight();
 		} else if (this._isTransitioning && this._visible) {
+			this._setCollapseHeight();
 			this._removeCollapseHeight();
 		}
```

Comment:
I can't comment on the correctness of this change, but the conditional might be more readable if you wrote it like this:

```javascript
if (this._isTransitioning) {
  this._setCollapseHeight();
  if (this._visible) {
    this._removeCollapseHeight();
  }
}
```

Obviously, readability is a bit subjective, but if you write it this way, the side-effect that is common to both branches is more obvious (ie. the `_setCollapseHeight` call) and also, the conditions under which it is called (ie. when `_isTransitioning`).

Reply:
Thanks for this, thanks @wincent!